test/test_shoppinglist.py

```diff
@@ -14,6 +14,7 @@ def test_get_shopping_list_valid(self, grocy):
         for item in shopping_list:
             assert isinstance(item, ShoppingListProduct)
             assert isinstance(item.id, int)
+            assert isinstance(item.done, int)
```

Comment:
Sorry, I'm a bit out-of-the-loop regarding grocy, is int the correct type for `done`. Sounds like it should be a boolean.